src/components/NotificationCenter/index.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { values } from "lodash";
+import { connect } from "react-redux";
+import Notification from "./Notification";
+import TodoList from "../TodoList";
+
+
+export function NotificationCenter ({notifications, dismissNotification, deactivateNotification}) {
+  return (
+    <div className="notification-center">
+      <ul>
+        {notifications.map(notification => {
+          return (
+            <li key={notification.id}>
+              <Notification
+                notification={notification}
+                onDismiss={() => dismissNotification(notification.id)}
+                onDeactivate={() => deactivateNotification(notification.id, notification.todo.id)}
+              />
+            </li>
+          )
+        })}
+      </ul>
+    </div>
+  )
+}
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    notifications: values(state.notifications).filter((notification) => notification.active)
+  }
+}
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+  return {
+    dismissNotification: (id) => {
+      dispatch({
+        type:'DISMISS_NOTIFICATION',
+        payload: {
+          id
+        }
+      })
+    },
+
+    deactivateNotification: (id, todoId, todoListId, updatedKeysInTodo) => {
```

Comment:
Here you declare the function `deactivateNotification` and you say it should receive 4 parameters when called.
So there is an inconsistency. In the end, precisely how many parameters does your function need ? 

To me it needs 3 parameters: `notificationId, todoListId, todoId`. 
- `notificationId` allows you to dimiss the notification
- `todoListId` allows you to find in which todoList is the todo that you want to deactivate
- `todoId` allows you to find the todo

You already know that you want to disable the todo right ? So `updatedKeysInTodo ` is an unnecessary parameter, you can remove it.